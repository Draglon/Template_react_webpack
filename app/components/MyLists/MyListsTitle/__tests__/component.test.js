import React from 'react';

import MyListsTitleComponent from '../component';

describe('<MyListsComponent />', () => {
  const component = shallow(<MyListsTitleComponent />);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
