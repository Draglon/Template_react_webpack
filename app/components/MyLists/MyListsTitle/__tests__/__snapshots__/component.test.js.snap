// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyListsComponent /> should match snapshot 1`] = `
<Row
  gutter={0}
>
  <Col
    offset={2}
    span={20}
  >
    <div
      className="top-margin"
    >
      <Title>
        My Lists 
        <Connect(CreateListModalContainer)
          icon="plus-circle"
        />
      </Title>
    </div>
  </Col>
</Row>
`;
